elastic.go

```diff
@@ -307,8 +328,16 @@ func (es *esClient) restoreDocuments(sb *s3Backend, keyPath, fileName string) er
 	}
 	defer fr.Close()
 
-	key := getKey(keyPath)
-	r, err := newDecryptor(key, fr)
+	privateKey, err := getPrivateKey(privateKeyPath, c4ghPassword)
+	if err != nil {
+		log.Error("Could not retrieve private key")
+
+		return err
+	}
```

Comment:
```suggestion
	if err != nil {
		return err
	}
```
No point of logging an error here since `err` will be propagated to main and logged as `fatal`